pkgs/development/libraries/geos/default.nix

```diff
@@ -1,10 +1,11 @@
 { lib
 , stdenv
+, callPackage
 , fetchurl
 , fetchpatch
 , cmake }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: rec {
```

Comment:
```suggestion
stdenv.mkDerivation (finalAttrs: {
```

Reply:
@sikmir , thank you for review. In PR #205770 I've got advice that it is actually recommended to keep `rec` even when using `stdenv.mkDerivation + finalAttrs`. Do you have some strong opinion on it ?

Reply:
OK, I see.